polaris/polaris/management/commands/poll_pending_deposits.py

```diff
@@ -135,9 +141,45 @@ def execute_deposits(cls):
                 "poll_pending_deposits() returned None. "
                 "Ensure is returns a list of transaction objects."
             )
+        distribution_accounts = {}
+        # Add the transactions that the anchor has been collecting signatures for
+        # but are now ready to submit.
+        # TODO: PR REVIEWERS, should we ask the anchor which transactions should
+        #  be added to the list of ready transactions or try to query them ourselves
+        #  like so?
+        #  We should definitely provide an integration function or explict
+        #  Transaction boolean column if the query below could in theory collect
+        #  transactions that are not ready. From my understanding, all Transaction
+        #  objects collected with the attribute values below _would_ be ready.
+        #  The idea is that anchors change the _status_ and _pending_signatures_
+        #  field to signal to Polaris that the object is ready.
+        ready_transactions.extend(
```

Comment:
@JakeUrban I can see two scenarios:
1. Polaris does this query and validates the envelope signatures (checks for duplicate signatures and sums up the weights). On this case, there's no integration function necessary and the Anchor must set `pending_signatures` to `True` to signal that the transaction is ready.
2. Polaris calls an integration function to get the ready transactions. On this case, Polaris doesn't check anything and just submits them. The Anchor would be responsible for all checks.

I don't see why Polaris would have an integration function and do the checks as well, it would add restriction on top of the integration.

Reply:
I agree, we can handle it.